cpp/src/io/orc/reader_impl.cu

```diff
@@ -695,22 +771,31 @@ void reader::impl::decode_stream_data(cudf::detail::hostdevice_2dvector<gpu::Col
 {
   const auto num_stripes = chunks.size().first;
   const auto num_columns = chunks.size().second;
+  thrust::counting_iterator<int, thrust::host_space_tag> col_idx_it(0);
+  thrust::counting_iterator<int, thrust::host_space_tag> stripe_idx_it(0);
 
   // Update chunks with pointers to column data
-  for (size_t i = 0; i < num_stripes; ++i) {
-    for (size_t j = 0; j < num_columns; ++j) {
-      auto& chunk            = chunks[i][j];
-      chunk.column_data_base = out_buffers[j].data();
-      chunk.valid_map_base   = out_buffers[j].null_mask();
-    }
-  }
+  std::for_each(stripe_idx_it + 0, stripe_idx_it + num_stripes, [&](auto stripe_idx) {
+    std::for_each(col_idx_it + 0, col_idx_it + num_columns, [&](auto col_idx) {
```

Comment:
Why the + 0?

Reply:
I might have missed while removing older code changes.